flixel/graphics/shaders/tiles/FlxTextured.hx

```diff
@@ -0,0 +1,61 @@
+package flixel.graphics.shaders.tiles;
+
+import flixel.graphics.shaders.FlxBaseShader;
+
+/**
+ * Default shader used by batcher for rendering textured quads.
+ */
+class FlxTextured extends FlxBaseShader
```

Comment:
I agree with Gama that more descriptive names would be better. For me the most important thing is consistency, so if the above class was to be renamed ```FlxColorShader``` than this one should be called ```FlxTextureShader```. You could also do ```FlxColoredShader``` and then this file would be called ```FlxTexturedShader```. But shorter names are also usually a better choice :-P